src/content/developers/docs/consensus-mechanisms/pow/index.md

```diff
@@ -3,76 +3,73 @@ title: Proof-of-work (PoW)
 
-Miners are incentivised to do this work on the main Ethereum chain. There is little incentive for a subset of miners to start their own chain – it undermines the system. Blockchains rely on having a single state as a source of truth. And users will always choose the longest or "heaviest" chain.
+Miners were incentivised to do this work on the main Ethereum chain. There was little incentive for a subset of miners to start their own chain – it undermines the system. Blockchains rely on having a single state as a source of truth.
 
-The objective of proof-of-work is to extend the chain. The longest chain is most believable as the valid one because it's had the most computational work done. Within Ethereum's PoW system, it's nearly impossible to create new blocks that erase transactions, create fake ones, or maintain a second chain. That's because a malicious miner would need to always solve the block nonce faster than everyone else.
+The objective of proof-of-work was to extend the chain. The longest chain was most believable as the valid one because it had the most computational work done to generate it. Within Ethereum's PoW system, it was nearly impossible to create new blocks that erase transactions, create fake ones, or maintain a second chain. That's because a malicious miner would have needed to always solve the block nonce faster than everyone else.
 
-To consistently create malicious yet valid blocks, you'd need over 51% of the network mining power to beat everyone else. You'd need a lot of computing power to be able to do this amount of "work". And the energy spent might even outweigh the gains you'd make in an attack.
+To consistently create malicious yet valid blocks, a malicious miner would have needed over 51% of the network mining power to beat everyone else. That amount of "work" requires a lot of expensive computing power and the energy spent might even have outweighed the gains made in an attack.
 
 ### Proof-of-work economics {#economics}
 
-Proof-of-work is also responsible for issuing new currency into the system and incentivizing miners to do the work.
+Proof-of-work was also responsible for issuing new currency into the system and incentivizing miners to do the work.
 
-Miners who successfully create a block get rewarded with two freshly minted ETH but no longer receive all the transaction fees, as the base fee gets burned, while the tip and block reward goes to the miner. A miner may also get 1.75 ETH for an uncle block. Uncle blocks are valid blocks created by a miner practically at the same time as another miner mined the successful block. Uncle blocks usually happen due to network latency.
+Miners who successfully create a block were rewarded with two freshly minted ETH and part of the transaction fees. 1.75 ETH was also paid for an uncle block. Uncle blocks are valid blocks created by a miner practically at the same time as another miner mined the successful block. Uncle blocks usually happen due to network latency.
```

Comment:
- "uncle" -> "ommer"
- Could mention Constantinople upgrade for historical reference on the 2 ETH reward. 
- Moved "1.75" from the beginning of the sentence.
- Added note about how one becomes canonical over the other
- Switched to past-tense

```suggestion
Since the [Constantinople upgrade](/history/#constantinople), miners who successfully create a block were rewarded with two freshly minted ETH and part of the transaction fees. Ommer blocks also compensated 1.75 ETH. Ommer blocks were valid blocks created by a miner practically at the same time as another miner created the canonical block, which was ultimately determined by which chain was built on top of first. Ommer blocks usually happened due to network latency.
```